releasenotes/notes/0.21/remove-hard-time-limit-vf2-be83830ecc71f72c.yaml

```diff
@@ -1,25 +0,0 @@
----
-upgrade:
```

Comment:
Did the associated changeset get reverted?

Reply:
This is actually just a git artifact. This was part of 0.21.0 not 0.21.1, but I accidentally committed it to the wrong path (see #8320) so it wasn't picked up as part of 0.21.0. The git history shows the file being created after 0.21.0. I just removed the note to ensure it wasn't picked up as part of 0.21.1. But for completeness I should add this to a note file already in 0.21.0. Let me do that really quickly

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8398/commits/4fb4391e096106a41a9c531112d4dbde1bbdbe55